File: std.js
const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout,
});

readline.question("Enter student name:",(name)=>{
    readline.question("Enter number of registered units:",(unitsStr)=>{
        const units=parseInt(unitsStr);
        let status = units > 7? "Overload - Approval Required":"Registration Accepted";

        console.log("NAME:" + name);
        console.log("Units registered:"+ units);
        console.log("Staus:"+status);

        readline.close();
    });
});

File: std.c
#include <stdio.h>


int main(){
    char name[50];
    int units;

    printf("Enter your name:");
    scanf("%49s", name);

    printf("Enter number of registered units:");
    scanf("%d", &units);

    char* status;
    if(units > 7){
        status = "Overload - Approval Required";
    }
    else{
        status = "Registration Accepted";
    }
    printf("NAME: %s\n",name);
    printf("UNITS: %d\n", units);
    printf("STATUS: %s\n", status);
    

    return 0;

}